<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JViewMainWindow</class>
 <widget class="QMainWindow" name="JViewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>461</width>
        <height>0</height>
       </size>
      </property>
      <property name="whatsThis">
       <string>Additional Image Options</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>coordinate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="coordinateLabel">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>[0,0]</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>value:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="colorLabel">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>[-----, ----- ,-----]</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sizeLabel">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>(0,0)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>scale:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="scaleLabel">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>exposure:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="exposureLabel">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>gamma:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gammaLabel">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>2.20</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="JImageView" name="viewWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_HDR"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save LDR</string>
   </property>
   <property name="toolTip">
    <string>Save LDR image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_HDR">
   <property name="text">
    <string>Save HDR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JImageView</class>
   <extends>QWidget</extends>
   <header>view.h</header>
   <container>1</container>
   <slots>
    <signal>gammaChanged(float)</signal>
    <signal>exposureChanged(float)</signal>
    <signal>scaleChanged(float)</signal>
    <signal>coordinateChanged(int,int)</signal>
    <slot>reset()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>viewWidget</sender>
   <signal>coordinateChanged(int,int)</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>setCoordinateLabel(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewWidget</sender>
   <signal>coordinateChanged(int,int)</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>setColorLabel(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewWidget</sender>
   <signal>exposureChanged(float)</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>setExposureLabel(float)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewWidget</sender>
   <signal>gammaChanged(float)</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>setGammaLabel(float)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewWidget</sender>
   <signal>scaleChanged(float)</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>setScaleLabel(float)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>openNewImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>viewWidget</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>saveLDRImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_HDR</sender>
   <signal>triggered()</signal>
   <receiver>JViewMainWindow</receiver>
   <slot>saveHDRImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCoordinateLabel(int,int)</slot>
  <slot>setColorLabel(int,int)</slot>
  <slot>setScaleLabel(float)</slot>
  <slot>setExposureLabel(float)</slot>
  <slot>setGammaLabel(float)</slot>
  <slot>openNewImage()</slot>
  <slot>saveLDRImage()</slot>
 </slots>
</ui>
